warp/logging/generateMarkdown.py

```diff
@@ -0,0 +1,93 @@
+import json
```

Comment:
The general code style for python code is to write function names and variables in "snake_case", as opposed to "camelCase"